processor/src/queues.ts

```diff
@@ -106,7 +108,7 @@ export async function addProjectToQueues({
     createJobs({
       giteaId,
       inputDir,
-      kitspaceYaml,
+      kitspaceYaml: setFallbackSummary(kitspaceYaml, repoDescription),
```

Comment:
```suggestion
      kitspaceYaml: kitspaceYaml.summary || repoDescription,

Reply:
This is the whole `kitspaceYaml` object not just the `summary`!

I could destructure the object and update `summary` but the function name makes it clear what is going on here.

Reply:
Ah, oops, sorry. I think updating summary inline is much more readable than abstracting this small bit of logic into a function though. 

```ts
projectKitspaceYaml.summary = projectKitspaceYaml.summary || repoDescription
```